src/components/ChatbotUi.js

```diff
@@ -72,21 +72,13 @@ const StyledChatBotUiContainer = styled.div`
   ${({ template }) => template === 'landing' && landingTemplateStyling};
 `;
 
-// the chatbot has 32px of left-side padding which makes the
-// loading skeleton have an alignment issue. it's great
-const SkeletonWrapper = styled.span`
-  padding-left: 32px;
-`;
-
 const LazyChatbot = lazy(() => import('mongodb-chatbot-ui'));
 
 const ChatbotUi = ({ template }) => {
   return (
     <StyledChatBotUiContainer data-testid="chatbot-ui" template={template}>
       {/* We wrapped this in a Suspense. We can use this opportunity to render a loading state if we decided we want that */}
-      <Suspense
-        fallback={<Skeleton wrapper={SkeletonWrapper} borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}
-      >
+      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}>
```

Comment:
would remove the width to have it adjust to mobile on load. not sure on setting an exact height..
```suggestion
      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} height={82} />}>
```